docs/ui-principles.rst

```diff
@@ -67,6 +67,8 @@ When writing interfaces use direct, clear and simple language. This is
 especially important as Warehouse caters to an international audience with
 varying proficiency in English. If you're unsure, `check the readability of
 your text <http://www.webpagefx.com/tools/read-able/>`_.
+Additionally for consistency of spellings, we prefer using American English
+spellings over British Engish.
```

Comment:
Let's just make this:

```
For consistency, we prefer using American English spellings over British English.
```

(note that you have "English" misspelled as "Engish" at the end here).

Reply:
Done!